src/mococrw/private/IOUtils.h

```diff
@@ -75,3 +82,21 @@ mococrw::AsymmetricPublicKey loadPubkeyFromFile(const std::string &filename)
         mococrw::AsymmetricPublicKey::readPublicKeyFromPEM>(filename);
 }
 
+mococrw::AsymmetricPrivateKey loadPrivkeyFromFile(const std::string &filename, const std::string &password)
+{
+    return openSSLObjectFromFile<mococrw::AsymmetricPrivateKey,
+        mococrw::AsymmetricPrivateKey::readPrivateKeyFromPEM>(filename, password);
+}
+
+void writePemToFile(const std::string &pem, const std::string &filename)
+{
+    std::ofstream file{filename, std::ofstream::out};
+    if (!file.good()) {
+        std::string errorMsg{"Cannot open file for writing key."};
```

Comment:
```suggestion
        std::string errorMsg{"Cannot open file for writing PEM."};
```

I guess this may also be useful for other PEM-encoded structs, not only keys, right? Also, the formatting can be improved here, since ideally the output would be:

`Cannot open file "filename" for writing: std::sterror(errno)`.